routes/users.js

```diff
@@ -826,16 +809,18 @@ router.delete_user = async function(user, action_owner_id, message){
         msg_destinations.push(user.email);
     }
 
-    let msg_del = CONFIG.message.deletion.join('\n').replace(/#UID#/g, user.uid).replace('#USER#', action_owner_id).replace('#MSG#', mail_message) + '\n' + CONFIG.message.footer.join('\n');
-    let msg_del_html = CONFIG.message.deletion_html.join('').replace(/#UID#/g, user.uid).replace('#USER#', action_owner_id).replace('#MSG#', mail_message) + '<br/>'+CONFIG.message.footer_html.join('<br/>');
-
-    let mailOptions = {
-        origin: MAIL_CONFIG.origin, // sender address
-        destinations:  msg_destinations, // list of receivers
-        subject: GENERAL_CONFIG.name + ' account deletion: ' + user.uid, // Subject line
-        message: msg_del, // plaintext body
-        html_message: msg_del_html // html body
-    };
+    try {
+        await utils.send_notif_mail({
+            'name': 'deletion',
+            'destinations': msg_destinations,
+            'subject': 'account deletion: ' + user.uid
+        }, {
+            '#UID#': user.uid,
+            '#MSG#': mail_message
```

Comment:
there was before variable *action_owner_id* , you do not set it here (unless you changed template to remove it?)

Reply:
https://github.com/genouest/genouestaccountmanager/pull/166/commits/e871307bafc84093fe6a8956a45d22e14e07ba0f